### PR TITLE
sl-160: Add exit button from site

### PR DESCRIPTION
**Pull request by jaroslawtt**
https://user-images.githubusercontent.com/87016333/230919839-a12c21ed-92e2-42f8-92b7-7231bd665e99.mp4

### CODE REVIEW

**Review by what1s1ove — CHANGES_REQUESTED** [review]
it should be fixed button (always be on the screen)
you don't to fetch a current site. Label of this button should be "Back to all sites"

**Comment by jaroslawtt** [comment]
> it should be fixed button (always be on the screen) you don't to fetch a current site. Label of this button should be "Back to all sites"

How I can get project id our site is associated with? As our path consist of "/projects/:projectId/sites", so to return back I ought to  possess this id.

**Comment by what1s1ove** [comment]
> > it should be fixed button (always be on the screen) you don't to fetch a current site. Label of this button should be "Back to all sites"
> 
> How I can get project id our site is associated with? As our path consist of "/projects/:projectId/sites", so to return back I ought to possess this id.

agreed. let's do it this way

File: frontend/src/packages/projects/projects-api.ts [hunk 1]
```diff
@@ -39,6 +39,26 @@ class ProjectsApi extends HttpApi {
     return await response.json<ProjectGetAllItemResponseDto>();
   }
 
+  public async getProject(parameters: {
```
Comment (what1s1ove) [inline frontend/src/packages/projects/projects-api.ts]:
```suggestion
  public async getById(parameters: {
```

File: frontend/src/pages/sites/sites.tsx [hunk 2]
```diff
@@ -46,15 +47,24 @@ const Sites: React.FC = () => {
             name: '',
           },
         }),
-      );
+      )
+        .unwrap()
+        .then(() => {
+          void dispatch(
```
Comment (what1s1ove) [inline frontend/src/pages/sites/sites.tsx]: call 2 action in parallel 
(sections and site)

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Comment by jaroslawtt** [comment]
https://user-images.githubusercontent.com/87016333/231073339-a94b9b2d-fa1f-4be5-aa00-dc372f5c85d4.mp4

File: frontend/src/pages/sites/sites.tsx [hunk 3]
```diff
@@ -20,10 +20,12 @@ import {
   useTitle,
 } from '~/libs/hooks/hooks.js';
 import { type ValueOf } from '~/libs/types/types.js';
+import { type ProjectGetAllItemResponseDto } from '~/packages/projects/libs/types/types.js';
```
Comment (what1s1ove) [inline frontend/src/pages/sites/sites.tsx]:
```suggestion
import { type ProjectGetAllItemResponseDto } from '~/packages/projects/projects.js';
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**